dotcom-rendering/src/web/components/FilterKeyEventsToggle.importable.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/react';
+import { from, neutral, remSpace } from '@guardian/source-foundations';
+import { ToggleSwitch } from '@guardian/source-react-components-development-kitchen';
+import { useEffect, useState } from 'react';
+
+const cssOverrides = css`
+	padding: ${remSpace[2]};
+
+	${from.phablet} {
+		padding-left: ${remSpace[5]};
+	}
+
+	${from.desktop} {
+		border-top: 1px solid ${neutral[86]};
+		padding-left: 0;
+	}
+`;
+
+export const FilterKeyEventsToggle = () => {
+	const [checked, setChecked] = useState(false);
+
+	const handleClick = () => {
+		setChecked(!checked);
+
+		const urlParams = new URLSearchParams(window.location.search);
+		urlParams.set('filterKeyEvents', checked ? 'false' : 'true');
+
+		window.location.search = urlParams.toString();
+		window.location.hash = 'maincontent';
+	};
+
+	useEffect(() => {
+		setChecked(window.location.search.includes('filterKeyEvents=true'));
+	}, [setChecked]);
+
+	return (
+		<ToggleSwitch
```

Comment:
putting these here so they don't get forgotten about
- [ ] label should be clickable 
- [ ] toggle should be tabbable and keyboard accessible 
- [ ] the animation is a bit off (green tick appears a bit too soon) and the toggle is not vertically aligned 
- [ ] in mobile view toggle should appear below the live feed header/separator rather than above it

Reply:
Think these are all resolved now